### DISCUSSION TITLE
MalformedJsonException: Expected name at line 1 column 556 path $.info.succ[1].

### DISCUSSION THREAD

**Discussion by Szpilman2** [UPPAALModelChecker/UPPAAL-Meta#228] [Q&A] [answered]
Hi everyone, 
I was working with my Model in UPPAAL when suddenly I get this Error Message, I checked my model and there isn't any problem or syntax error, and I don't know why I get this exception. could you please help me?

best regards. 
Peyman

![error](https://github.com/UPPAALModelChecker/UPPAAL-Meta/assets/35931007/d089e7dd-6ad4-4f3e-a524-da5b27cde6eb)

also this is the xml file content at line 556.
![xmlContent](https://github.com/UPPAALModelChecker/UPPAAL-Meta/assets/35931007/b7d3fee7-0ed9-42ca-8d5c-a0f4feab1287)

**Comment by mikucionisaau** [comment]
The error tells me that there was an issue with computing a transition successors, so the issue is somewhere within one of the simulators. Either the model has something special that crashes the engine or engine responds in a format that GUI does not expect.
Anyway, issues like that need to be replicated on our side to be able to fix it.
I would appreciate if you could attach the model here or send it to us.

**Reply by Szpilman2** [reply]
Hi, thank you so much for your response. 
this is the model:
[MalformedModel.zip](https://github.com/UPPAALModelChecker/UPPAAL-Meta/files/13997525/MalformedModel.zip)


also it is a screen recording which shows the error:
https://github.com/UPPAALModelChecker/UPPAAL-Meta/assets/35931007/56719bb0-5611-45de-a3bc-a63ce78ec102


best regards

**Reply by mikucionisaau** [reply, marked as answer]
Thank you!
I can reproduce the issue with Uppaal-5.0.0, but not with Uppaal-5.1.0-beta5.
The model has lot's of deadlock transitions and there were some fixes related to that, so perhaps it's been fixed there.
Could you please try 5.1.0-beta5?

**Reply by Szpilman2** [reply]
Dear Marius,

I wanted to express my gratitude for your assistance. I've tested UPPAAL 5.1 beta 5, and I'm pleased to report that the previous errors are no longer present. Thank you so much for your help.

Best regards,
Peyman

**Reply by mikucionisaau** [reply]
Thank you Peyman for your reports, we really appreciate them.
